### DISCUSSION TITLE
Math formulas not rendering, sometimes

### DISCUSSION THREAD

**Discussion by AtmoFX** [community/community#82399] [Discussions]
### Select Topic Area

Bug

### Body

Some math formulas do not render.<br/>
Below are 3 cases of some formulas being rendered vs some other formulas NOT being rendered when they are, essentially, the same.

Note: Contrary to https://github.com/orgs/community/discussions/17325, this occurs outside of lists.

-----

Case 1:

1.1 renders if the superscript only has 1 digit:
`F^5_{10}[2] = F^5_{10}[3] + F^5_{10}[5]`

$$
F^5_{10}[2] = F^5_{10}[3] + F^5_{10}[5]
$$

1.2 renders if the subscript comes first.
`F_{10}^{15}[2] = F_{10}^{15}[3] + F_{10}^{15}[5]`

$$
F_{10}^{15}[2] = F_{10}^{15}[3] + F_{10}^{15}[5]
$$

1.3 renders if expressed as in-line formula:
`F^{15}_{10}[2] = F^{15}_{10}[3] + F^{15}_{10}[5]`

$`
F^{15}_{10}[2] = F^{15}_{10}[3] + F^{15}_{10}[5]
`$

Does not render:
`F^{15}_{10}[2] = F^{15}_{10}[3] + F^{15}_{10}[5]`

$$
F^{15}_{10}[2] = F^{15}_{10}[3] + F^{15}_{10}[5]
$$

-----

Case 2:

Renders; the last F is formatted as \mathscr{F}:
`F_{12}^{10}[1,4] = F_{12}^{10}[1,10] + \mathscr{F}_{12}^{10}[2,5]`

$$
F_{12}^{10}[1,4] = F_{12}^{10}[1,10] + \mathscr{F}_{12}^{10}[2,5]
$$

Does not render if the \mathscr{F} is the second one:
`F_{12}^{10}[1,4] = \mathscr{F}_{12}^{10}[1,10] + F_{12}^{10}[2,5]`

$$
F_{12}^{10}[1,4] = \mathscr{F}_{12}^{10}[1,10] + F_{12}^{10}[2,5]
$$

Does not render if the \mathscr{F} is the first one:
`\mathscr{F}_{12}^{10}[1,4] = F_{12}^{10}[1,10] + F_{12}^{10}[2,5]`

$$
\mathscr{F}_{12}^{10}[1,4] = F_{12}^{10}[1,10] + F_{12}^{10}[2,5]
$$

-----

Case 3
Renders: The subscript of the sum is `i < k`, with spaces:

$$
\sum_{i < k} a_i 
$$

Does not render: the subscript of the sum is `i<k`.<br/>This fails with an error message, i.e. differently from the above 2 cases; the message gives no clue about what makes it fail though:

$$
\sum_{i<k} a_i 
$$

**Comment by Nqvinh7603** [comment]
response #1
